results.tex

```diff
@@ -1,10 +1,24 @@
 \subsection{Results}
 By using the approach described in the previous section, the new mapper grew capable of providing critical features for medical and scientific computing. This section describes each of these features in sufficient detail in the following text.
 
-\subsubsection{Jittering}
+\newcommand{\ignore}[1]{}
+
+\subsubsection{Stochastic Jittering}
+Because ray-casting effectively samples a discrete signal (voxel values along the ray trajectory), the distance between those sampling points heavily influences how accurately the volume data is represented.  Due to well established limitations described by the sampling theorem, low sampling rates may result in aliasing effects 
+often referred to as wood-grain artifacts in the context of volume rendering \ignore{\cite{RTVG_jittering}}. Reducing the distance between samples neutralizes these artifacts but takes an important toll on performance.
+
+
+The mapper supports stochastic jittering, which is an alternative technique to counteract wood-grain artifacts by adding a random offset to the rays in the viewing direction thereby breaking the coherence between neighboring fragments which causes the aliased patterns become apparent (Figure~\ref{fig:jittering}).  Jittering is implemented by creating a random noise texture (using \texttt{vtkPerlinNoise}) and applying the offset to the ray's starting point, thus has a much lower performance penalty than reducing the sampling distance.
```

Comment:
I will need you to tell me again where I can get one of the datasets to create a nice figure for this.